lib/enhance.js

```diff
@@ -3,7 +3,8 @@
   var Enhance, _;
 
   _ = require('./lodash-ext');
-  var lodashExt = _.noConflict();
+
+  _.noConflict();
```

Comment:
nah it puts itself on the window but the assignment on line 5 is local because coffeescript